### PR TITLE
downsample ppg

### PR DESCRIPTION
**Pull request by Andrey1994**
Signed-off-by: Andrey Parfenov <a1994ndrey@gmail.com>

### CODE REVIEW

File: OpenBCI_GUI/W_NovaAux.pde [hunk 1]
```diff
@@ -266,14 +270,25 @@ abstract class AuxReadBar{
...
+            for (int i = 0; i < totalPoints; i+=downscale) {
+                downscaledData.add(allData.get(i));
+            }
+        }
         
-        double max = allData.get(0)[channel];
+        double max = downscaledData.get(0)[channel];
         double min = max;
 
         for (int i=0; i < nPoints; i++) {
```
Comment (retiutut) [inline OpenBCI_GUI/W_NovaAux.pde:289]: Maybe these two `for` loops can be refactored to just one?

File: OpenBCI_GUI/BoardNovaXR.pde [hunk 2]
```diff
@@ -2,7 +2,7 @@ import brainflow.*;
 
 import org.apache.commons.lang3.ArrayUtils;
 
-final boolean novaXREnabled = false;
+final boolean novaXREnabled = true;
```
Comment (retiutut) [inline OpenBCI_GUI/BoardNovaXR.pde:5]: This needs to be set back to `false` before merging

**Review by retiutut — CHANGES_REQUESTED** [review]

**Comment by retiutut** [comment]
Waiting for results from internal test on this fix...

**Comment by Andrey1994** [comment]
tested with real device and looks fine

**Pull request merged**